docs/intro.rst

```diff
@@ -28,6 +28,7 @@ pages for each source:
 We also publish SQLite databases containing relatively pristine versions of our more
```

Comment:
The intro in the intro page feels duplicative with the intro on the readme page. I think we should just have one or the other, not both.